src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -306,30 +306,30 @@ public bool SkipCompilerExecution
             get { return _store.GetOrDefault(nameof(SkipCompilerExecution), false); }
         }
 
-        public ITaskItem[] Sources
+        public ITaskItem[]? Sources
         {
             set
             {
                 if (UsedCommandLineTool)
                 {
-                    NormalizePaths(value);
+                    NormalizePaths(value!); // PROTOTYPE(NullableDogfood): value set should be non-null
```

Comment:
💭 I believe this is going to be a common request. Technically the situation occurs many times in this file but this is the only one that catches it.